lib/PuppeteerSharp/Helpers/FlexibleStringEnumConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace PuppeteerSharp.Helpers
+{
+    internal class FlexibleStringEnumConverter : StringEnumConverter
```

Comment:
I'm guessing this file exists for the `other` case

Reply:
Exactly. I tried with `StringEnumConverter` but it failed when it got the type  `background_page`, and I was unable to find the complete list of types.